src/sentry/south_migrations/0455_create_global_searches.py

```diff
@@ -0,0 +1,1269 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+from sentry.models.savedsearch import DEFAULT_SAVED_SEARCHES
```

Comment:
I'd vendor this variable to not rely on the import. This seems like something that'd be very likely to be moved or changed, in which case, migrations will be broken from a bad import. It's safer to just copy/paste the data in here so it's a snapshot in time.